sdks/python/apache_beam/io/fileio_test.py

```diff
@@ -631,7 +660,16 @@ def test_write_text_bzip2_file(self):
     with bz2.BZ2File(self.path, 'r') as f:
       self.assertEqual(f.read().splitlines(), self.lines)
 
-  def test_write_text_bzip2_file_auto(self):
+    nspace, dd = make_nspace_display_data(sink)
+    expected_items = [
+        DisplayDataItem('{}{}'.format(self.path, '-SSSSS-of-NNNNN'),
```

Comment:
I looked at the code for the sources and couldn't find where the file name is created. It seems that it's just sent to the dataflow service. I think this should be tested in an integration test. - But I've added an extra test case to keep track of this in case it changes.